### PR TITLE
Refactor/MZ-626

### PR DESCRIPTION
**Pull request by qnen**
# Midaz Pull Request Checklist

## Pull Request Type
[//]: # (Check the appropriate box for the type of pull request.)

- [ ] Auth
- [ ] Infra
- [x] Ledger
- [ ] Mdz
- [x] Transaction
- [ ] Pipeline
- [ ] Documentation

## Checklist
Please check each item after it's completed.

- [x] I have tested these changes locally.
- [x] I have updated the documentation accordingly.
- [x] I have added necessary comments to the code, especially in complex areas.
- [x] I have ensured that my changes adhere to the project's coding standards.
- [x] I have checked for any potential security issues.
- [x] I have ensured that all tests pass.
- [ ] I have updated the version appropriately (if applicable).
- [x] I have confirmed this code is ready for review.

## Additional Notes
[//]: # (Add any additional notes, context, or explanation that could be helpful for reviewers.)
## Obs: Please, always remember to target your PR to develop branch instead of main.

### CODE REVIEW

**Review by maxwelbm — APPROVED** [review]

File: common/net/http/headers.go [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package http
 
 const (
+	headerMidazID       = "Midaz-ID"
```
Comment (MartinezAvellan) [inline common/net/http/headers.go:4]: what is the correct name? **midaz-ID**, **midaz-Id** or **midaz-id**???? please use only one.

File: common/constant/metadata.go [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package constant
+
+const (
+	MDMidazID       = "midaz_id"
```
Comment (MartinezAvellan) [inline common/constant/metadata.go:4]: what is the correct name? **midaz-ID**, **midaz-Id** or **midaz-id**???? please use only one.

File: postman/MIDAZ.postman_collection.json [hunk 3]
```diff
@@ -2549,7 +2865,14 @@
 					],
 					"request": {
 						"method": "GET",
-						"header": [],
+						"header": [
+							{
+								"key": "Midaz-Id",
```
Comment (MartinezAvellan) [inline postman/MIDAZ.postman_collection.json]: what is the correct name? **midaz-ID**, **midaz-Id** or **midaz-id**???? please use only one.

File: postman/MIDAZ.postman_collection.json [hunk 4]
```diff
@@ -2500,7 +2809,14 @@
 					],
 					"request": {
 						"method": "POST",
-						"header": [],
+						"header": [
+							{
+								"key": "Midaz-Id",
```
Comment (MartinezAvellan) [inline postman/MIDAZ.postman_collection.json]: what is the correct name? **midaz-ID**, **midaz-Id** or **midaz-id**???? please use only one.

File: postman/MIDAZ.postman_collection.json [hunk 5]
```diff
@@ -2453,7 +2755,14 @@
 					"name": "Portfolios -> Operation by id",
 					"request": {
 						"method": "GET",
-						"header": [],
+						"header": [
+							{
+								"key": "Midaz-Id",
```
Comment (MartinezAvellan) [inline postman/MIDAZ.postman_collection.json]: what is the correct name? **midaz-ID**, **midaz-Id** or **midaz-id**???? please use only one.

File: postman/MIDAZ.postman_collection.json [hunk 6]
```diff
@@ -2429,7 +2724,14 @@
 					"name": "Portofolios -> Operations",
 					"request": {
 						"method": "GET",
-						"header": [],
+						"header": [
+							{
+								"key": "Midaz-Id",
```
Comment (MartinezAvellan) [inline postman/MIDAZ.postman_collection.json]: what is the correct name? **midaz-ID**, **midaz-Id** or **midaz-id**???? please use only one.

File: postman/MIDAZ.postman_collection.json [hunk 7]
```diff
@@ -2404,7 +2692,14 @@
 					"name": "Accounts -> Operation by id",
 					"request": {
 						"method": "GET",
-						"header": [],
+						"header": [
+							{
+								"key": "Midaz-Id",
```
Comment (MartinezAvellan) [inline postman/MIDAZ.postman_collection.json]: what is the correct name? **midaz-ID**, **midaz-Id** or **midaz-id**???? please use only one.

File: postman/MIDAZ.postman_collection.json [hunk 8]
```diff
@@ -2380,7 +2661,14 @@
 					"name": "Account -> Operations",
 					"request": {
 						"method": "GET",
-						"header": [],
+						"header": [
+							{
+								"key": "Midaz-Id",
```
Comment (MartinezAvellan) [inline postman/MIDAZ.postman_collection.json]: what is the correct name? **midaz-ID**, **midaz-Id** or **midaz-id**???? please use only one.

File: postman/MIDAZ.postman_collection.json [hunk 9]
```diff
@@ -2345,7 +2619,14 @@
 					],
 					"request": {
 						"method": "PATCH",
-						"header": [],
+						"header": [
+							{
+								"key": "Midaz-Id",
```
Comment (MartinezAvellan) [inline postman/MIDAZ.postman_collection.json]: what is the correct name? **midaz-ID**, **midaz-Id** or **midaz-id**???? please use only one.

File: postman/MIDAZ.postman_collection.json [hunk 10]
```diff
@@ -2306,7 +2573,14 @@
 					"name": "Transactions",
 					"request": {
 						"method": "GET",
-						"header": [],
+						"header": [
+							{
+								"key": "Midaz-Id",
```
Comment (MartinezAvellan) [inline postman/MIDAZ.postman_collection.json]: what is the correct name? **midaz-ID**, **midaz-Id** or **midaz-id**???? please use only one.

File: postman/MIDAZ.postman_collection.json [hunk 11]
```diff
@@ -2283,7 +2543,14 @@
 					"name": "Transaction by id",
 					"request": {
 						"method": "GET",
-						"header": [],
+						"header": [
+							{
+								"key": "Midaz-Id",
```
Comment (MartinezAvellan) [inline postman/MIDAZ.postman_collection.json]: what is the correct name? **midaz-ID**, **midaz-Id** or **midaz-id**???? please use only one.

File: postman/MIDAZ.postman_collection.json [hunk 12]
```diff
@@ -2250,7 +2503,14 @@
 					],
 					"request": {
 						"method": "PATCH",
-						"header": [],
+						"header": [
+							{
+								"key": "Midaz-Id",
```
Comment (MartinezAvellan) [inline postman/MIDAZ.postman_collection.json]: what is the correct name? **midaz-ID**, **midaz-Id** or **midaz-id**???? please use only one.

File: postman/MIDAZ.postman_collection.json [hunk 13]
```diff
@@ -1987,6 +2234,12 @@
 										"value": "300",
 										"description": "seconds",
 										"type": "text"
+									},
+									{
+										"key": "Midaz-Id",
+										"value": "{{$randomUUID}}",
```
Comment (MartinezAvellan) [inline postman/MIDAZ.postman_collection.json:2240]: what is the correct name? **midaz-ID**, **midaz-Id** or **midaz-id**???? please use only one.

File: postman/MIDAZ.postman_collection.json [hunk 14]
```diff
@@ -1921,7 +2161,14 @@
 							],
 							"request": {
 								"method": "POST",
-								"header": [],
+								"header": [
+									{
+										"key": "Midaz-Id",
+										"value": "{{$randomUUID}}",
```
Comment (MartinezAvellan) [inline postman/MIDAZ.postman_collection.json:2167]: what is the correct name? **midaz-ID**, **midaz-Id** or **midaz-id**???? please use only one.

**Review by MartinezAvellan — CHANGES_REQUESTED** [review]

**Review by MartinezAvellan — APPROVED** [review]

**Pull request merged**